File: solutions/python/raindrops/raindrops.py
def convert(number):
    converted = ''
    if number % 3 == 0:
        converted = 'Pling'
    if number % 5 == 0:
        converted += 'Plang'
    if number % 7 == 0:
        converted += 'Plong'
    return converted if converted else str(number)

File: solutions/typescript/raindrops/raindrops.ts
export function convert(drops: number): string {
  let converted = ''

  if (drops % 3 === 0) {
    converted += 'Pling'
  }
  if (drops % 5 === 0) {
    converted += 'Plang'
  }
  if (drops % 7 === 0) {
    converted += 'Plong'
  }

  if (!converted) {
    return drops.toString()
  }
  
  return converted
}